app/views/spree/admin/orders/invoice.html.haml

```diff
@@ -30,13 +30,14 @@
         &nbsp;
       %td{ :align => "right" }
         %strong= "#{t('.to')}: #{@order.ship_address.full_name}"
-        - if @order.customer.code.present?
+        - if @order.andand.customer.andand.code.present?
```

Comment:
Whilst testing I found a few orders that caused a fatal error here. After looking into it I found it wasn't caused by this PR, and those same orders threw a fatal error when making a regular invoice PDF.

Reply:
Possibly due to legacy data?

Reply:
99% sure. It must be invalid data in DB :boom: . Could tell why do they fail and which ones? I'd like to have an issue to fix them.

Reply:
Note: customer can be `nil` on an order, and code can be `nil` on a customer. We discussed implementing nil objects, but decided it might not be worth the effort for this one line.